CppCoreGuidelines.md

```diff
@@ -14196,6 +14198,30 @@ Really, "all bets are off" if you get a data race.
 Actually, it can be worse still: by looking at the generated code you might be able to determine where the stray jump will go for a given value;
 this can be a security risk.
 
+##### Example, better
+A simple strategy to avoid such global races and to guarantee (at least) local code-correctness is:
+1) Make a local copy.
+2) Validate.
+3) Make further processing on the local only.
+That works, because every thread has it's own (functional) stack containing all (thread-)local variables.
+
+    unsigned val; //  global value
+    void f()
+    {
+        auto loc = val;
```

Comment:
data race absent additional synchronization, as already mentioned.
Explicit bounds check doesn't avoid a data race, so is out of place in the rule about avoiding data races.